src/ui/dialogs.py

```
@@ -164,20 +195,43 @@ def display_record_details(self):
+                try:
+                    import requests
+                    # Disable TLS certificate verification to avoid failures in bundled executables
+                    resp = requests.get(path, timeout=5, verify=False)
+                    if resp.status_code == 200:
+                        pixmap = QPixmap()
+                        pixmap.loadFromData(resp.content)
+                        label.setPixmap(pixmap.scaled(label.size(), Qt.KeepAspectRatio, Qt.SmoothTransformation))
+                        label.setText("")
+                    else:
+                        label.setText(f"No {img_type.replace('_', ' ').title()}")
+                except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)